--- a/lib/rb/spec/types_spec.rb	2018-09-12 21:07:38.954646347 +0200
+++ b/lib/rb/spec/types_spec.rb	2018-09-12 20:26:35.269937386 +0200
@@ -97,9 +97,9 @@
     end
 
     it "should give the Thrift::TypeError a readable message" do
-      msg = "Expected Types::STRING, received Fixnum for field foo"
+      msg = "Expected Types::STRING, received Integer for field foo"
       lambda { Thrift.check_type(3, {:type => Thrift::Types::STRING}, :foo) }.should raise_error(Thrift::TypeError, msg)
-      msg = "Expected Types::STRING, received Fixnum for field foo.element"
+      msg = "Expected Types::STRING, received Integer for field foo.element"
       field = {:type => Thrift::Types::LIST, :element => {:type => Thrift::Types::STRING}}
       lambda { Thrift.check_type([3], field, :foo) }.should raise_error(Thrift::TypeError, msg)
       msg = "Expected Types::I32, received NilClass for field foo.element.key"
